content/operate/kubernetes/reference/redis_enterprise_remote_cluster_api.md

```diff
@@ -0,0 +1,167 @@
+---
+title: RedisEnterpriseRemoteCluster API Reference
+alwaysopen: false
+categories:
+- docs
+- operate
+- kubernetes
+linkTitle: RERC API
+weight: 30
+---
+
+API groups:
+
+
+- [app.redislabs.com/v1alpha1](#appredislabscomv1alpha1)
+
+
+
+
+# app.redislabs.com/v1alpha1
+
+
+
+
+RedisEntepriseRemoteCluster represents a remote participating cluster.
+
+<table>
```

Comment:
Are the spec and status descriptions intentionally left out?

Reply:
I'll make a note of that for when I do copyediting.